spec/forms/provider_interface/interview_wizard_spec.rb

```diff
@@ -157,17 +157,39 @@
   end
 
   describe '#date_and_time' do
-    let(:time) { '4:30pm' }
     let(:day) { '20' }
     let(:month) { '2' }
     let(:year) { '2022' }
+    let(:valid_times) do
```

Comment:
This might be a little more descriptive as:
```
let(:time_inputs_and_expectations) do
  [
    { input: '12:30pm', expected_hour: '12', expected_minute: '30' },
    ...
  ]
end
```
What do you think?

Reply:
yea, that's better